test/extended/openstack/egressip.go

```diff
@@ -34,6 +36,7 @@ const (
 )
 
 var _ = g.Describe("[sig-installer][Suite:openshift/openstack][egressip] An egressIP", func() {
+	defer g.GinkgoRecover()
```

Comment:
Nice. Just to confirm that my understanding is correct, this is to recover any failure outside of a ginkgo node? In this case, this is just the call to `exutil.NewCLI()` below?

Reply:
That function is documented to recover from panics set out by Ginkgo itself in goroutines. However, as far as I know it will recover any panic, and that's why I tend to put it in every node. However, as you note there is no compelling reason to add it now; I am removing this line and we'll react if we ever see panics in tests.